Makefile

```diff
@@ -2,52 +2,30 @@
 
 all: deps compile
 
-compile: rebar
-	./rebar compile
+compile: rebar3
+	./rebar3 compile
 
-deps: rebar
-	./rebar get-deps
+deps: rebar3
+	./rebar3 get-deps
 
-clean: rebar
-	./rebar clean
+clean: rebar3
+	./rebar3 clean
 
-test-deps: rebar
-	./rebar -C rebar.test.config get-deps
+test-deps: rebar3
+	./rebar3 get-deps
 
-test-compile: rebar test-deps
-	./rebar -C rebar.test.config compile
+test-compile: rebar3 test-deps
+	./rebar3 compile
 
 test: test-compile
-	./rebar -C rebar.test.config skip_deps=true eunit
+	./rebar3 eunit
 
 coverage-report: $(shell ls -1rt `find .eunit -type f -name \*.coverdata 2>/dev/null` | tail -n1)
 	erl -noshell -pa ebin deps/*/ebin -eval 'ecoveralls:travis_ci("$?"), init:stop()'
```

Comment:
This will not work, since there will be no `deps` directory anymore. `make coverage-report` is used in `.travis.yml`. Not sure what to do with it yet...

Reply:
This could be handled in a similar way as in MongooseIM, see:
* `.travis.yml` - https://github.com/esl/MongooseIM/blob/master/.travis.yml#L49
*  `rebar.config` - https://github.com/esl/MongooseIM/blob/master/rebar.config#L173-L177